charts/backstage/Chart.yaml

```diff
@@ -27,3 +27,6 @@ dependencies:
     name: postgresql
     repository: https://charts.bitnami.com/bitnami
     version: 11.x.x
+annotations:
+  artifacthub.io/changes: |
+    - "[enhancement]: Adds Backstage ServiceAccount"
```

Comment:
since we are not publishing to artifacthub it would mean we'll loose the changelogs between different releases, right?

Reply:
@vinzscam Ah, my bad! I thought for some reason we were publishing to ArtifactHub - I must be confusing my projects 😅  I shall remove the `artifacthub.io/changes`. Or do you think it's worth us publishing? Or is this something that you feel should be done when the moves over into the `backstage` org?